src/models/DefaultFilters.js

```diff
@@ -0,0 +1,40 @@
+'use strict'
+
+const ElectronConfig = require('electron-config')
+const configName = process.env.NODE_ENV === 'test' ? 'config-test' : 'config'
+const storage = new ElectronConfig({ name: configName })
+const Filter = require('./Filter')
+const LastFilter = require('./LastFilter')
+const DEFAULT_FILTER_KEY = 'default-filters'
+
+const DEFAULT_FILTERS = {
+  'My Issues': 'author:login is:open sort:updated-desc',
+  'My Comments': 'commenter:login sort:updated-desc',
+  'My Assignments': 'assignee:login sort:updated-desc is:open',
+  'My Mentions': 'mentions:login is:open sort:updated-desc',
+  'My Popular Issues': 'author:login is:open interactions:>5 sort:updated-desc',
+}
+
+class DefaultFilters {
+  constructor(login) {
+    this.login = login
+  }
+
+  addDefaults() {
+    if (storage.has(DEFAULT_FILTER_KEY)) {
+      return
+    }
+
+    const filterNames = Object.keys(DEFAULT_FILTERS)
+    filterNames.forEach(name => {
+      const filter = new Filter(name)
+      const value = DEFAULT_FILTERS[name].replace(/login/g, this.login)
```

Comment:
I could see a filter being like `-author:cheshire137 commenter:cheshire137` to show things I've commented on but didn't open myself, for example.